src/components/subComponents/DataDownload.vue

```diff
@@ -165,6 +173,62 @@
     .then((response: any) => response.data)
     .then((rs) => (factories = rs))
     .then(loadData);
+
+  function interceptPageNavigation(): void {
+    dialog.open(ConfirmationDialog, {
+      data: {
+        message: t('monitoringData.dialog.msg.downloadStudyData'),
+        cancelBtn: t('monitoringData.dialog.waitForDownload'),
+        approveBtn: t('monitoringData.dialog.navigatePage')
+      },
+      props: {
+        header: t('monitoringData.dialog.header.downloadStudyData'),
+        style: {
+          width: '50vw',
+        },
+        breakpoints: {
+          '960px': '75vw',
+          '640px': '90vw',
+        },
+        modal: true,
+        draggable: false,
+      },
+      onClose: (options) =>{
+        if (options?.data) {
+          if (pendingRoute.value) {
+            isDownloadDataLoading.value = false
+            router.push(pendingRoute.value)
+            pendingRoute.value = null
+          }
+        } else {
+          pendingRoute.value = null
+        }
```

Comment:
Kann vereinfacht werden
```suggestion
        if (options?.data) {
          if (pendingRoute.value) {
            isDownloadDataLoading.value = false
            router.push(pendingRoute.value)
          }
        }
  
        pendingRoute.value = null
        
```